lib/simperium/bucket-store.js

```diff
@@ -67,7 +67,7 @@ BucketStore.prototype.get = function(id, callback) {
 	});
 };
 
-BucketStore.prototype.update = function(id, data, callback) {
+BucketStore.prototype.update = function(id, data, isIndexing, callback) {
```

Comment:
I didn't see any changes to where we actually call this function. I guess it's just good practice to declare it to match the library call?